### DISCUSSION TITLE
Is there a compiled download for this? if yes where can I find it?

### DISCUSSION THREAD

**Discussion by GenericDevUsername** [AtriusX/Waystones#24] [General]
See title

**Comment by AtriusX** [comment]
Hello, no releases were previously published since interest in the project has currently been fairly limited. I've taken the liberty to publish a release of the current repository [here](https://github.com/AtriusX/Waystones/releases/tag/0.1.0). If you have any ideas or questions related to the current state of that release or the plugin in general you can discuss it [here](https://github.com/AtriusX/Waystones/discussions/25). Thanks!

**Reply by GenericDevUsername** [reply]
Thanks, I appreciate it.
And as to the limited interest in the project. It’s probably because it hasn’t been advertised/posted on Mc-Market/SpigotMC at any time.

**Reply by AtriusX** [reply]
That is definitely the case. The project is mostly done but remains sort of in on/off development as referenced by #22. If you have any ideas to provide for that I would be pleased to hear them.

**Reply by GenericDevUsername** [reply]
Sure, I’ll have a read

**Reply by GenericDevUsername** [reply]
I’ve made a post about my ideas on the topic at https://github.com/AtriusX/Waystones/discussions/22#discussioncomment-720776

**Reply by Superola86** [reply]
You should release this on spigot, is a great idea.

**Reply by GenericDevUsername** [reply]
This is exactly what I suggested, it's such a cool concept

**Reply by AtriusX** [reply]
This will probably happen sometime in the future. Personally I wanted to iron out any remaining problems I could find before putting a release out, but it might be better to just release it 'as-is' once a few of the remaining issues are sorted out.

If either of you have any input or desire to help with testing I encourage you to look at #27 or #30!

**Reply by AtriusX** [reply]
As of now the plugin is up on spigot! I posted a small discussion for it at #34.

**Reply by GenericDevUsername** [reply]
that's great, hopefully it will gain some more traction now

**Reply by GenericDevUsername** [reply]
![WarpKey](https://user-images.githubusercontent.com/59979897/124457426-b4e08780-dd83-11eb-8c13-637561ba67b9.png)
For the warp key recipe in your spigot description

**Reply by AtriusX** [reply]
How come the key is a tripwire hook? I'm pretty sure the key itself is a compass and isn't customizable. If you can update it I'll add it to the page. I appreciate it otherwise though!

**Reply by GenericDevUsername** [reply]
oops my bad 
ill update it now

**Reply by GenericDevUsername** [reply]
![JGQnbf](https://user-images.githubusercontent.com/59979897/124510978-e6ca0c00-ddcc-11eb-8666-0169c8291537.png)

**Reply by AtriusX** [reply]
Great, thanks!

**Reply by GenericDevUsername** [reply]
No problem :)
